packages/core/src/data-editor/data-editor.tsx

```diff
@@ -467,6 +467,12 @@ export interface DataEditorProps extends Props {
      * @group Selection
      */
     readonly gridSelection?: GridSelection;
+    /**
+     * Scroll position configuration when using {@link gridSelection}.
+     * See "Controlled selection" example for details.
+     * @group Selection
+     */
+    readonly gridSelectionScrollOptions?: GridSelectionScrollOptions;
```

Comment:
I really like this option, I think we need to workshop the name a bit. Not quite sure I have a better alternative yet... Did you have any other ideas?

Reply:
maybe a shorter more generic name like: "gridSelectionScroll" works better.

Reply:
`focusScrollBehavior` `focusBehavior` `gridSelectionBehavior`

One of the things about this is it actually only reflects the `gridSelection.current.cell`

Reply:
I prefer ```gridSelectionBehavior```

Reply:
approved

Reply:
I still need to do an audit of how this interacts with all the other options, would be helpful to break down how this corresponds with the paddingX/Y options etc

Reply:
@jassmith anything oustanding on my end for this to get merged?